api/src/controllers/auth-controller.ts

```diff
@@ -68,46 +72,46 @@ export const jwtLoginController = async (req: Request, res: Response) => {
 
   if (!user) {
     return res.status(401).json({
+      success: false,
       isLoggedIn: false,
-      message: 'User failed to authenticate.',
+      message: 'Invalid credentials',
```

Comment:
Do we need these two messages? Apart from being more descriptive, do we use them in the front-end?

Can we make the response from this endpoint consistient